build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group 'org.logstash.beats'
+version "${new File("VERSION").text.trim()}"
+
+description = "Beats/Lumberjack Netty implementation"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.hamcrest:hamcrest-library:1.3'
+
+    compile 'org.bouncycastle:bcprov-jdk15on:1.54'
+    compile 'org.bouncycastle:bcpkix-jdk15on:1.54'
+    compile 'io.netty:netty-all:4.1.1.Final'
+    compile 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork17'
+    compile 'org.apache.logging.log4j:log4j-api:2.6.1'
+    compile 'org.apache.logging.log4j:log4j-core:2.6.1'
+    compile 'org.lucee:javassist:3.9.0.GA'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.7.4'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.7.4'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.4'
+    compile 'com.fasterxml.jackson.module:jackson-module-afterburner:2.7.4'
```

Comment:
2.7.5 is uptodate, I think the important part is not to have other plugins with different versions in your classpath